bird.js

```diff
@@ -26,12 +26,17 @@ class Bird {
   }
 
   up() {
-    this.velocity += this.lift;
+    if (this.delay)
```

Comment:
Isn't this whole `this.delay` variable kind of irrelevant? Sorry if  I miss something but it seems like you could delete the whole if-statement and just leave line 33.